### PR TITLE
fozzie-components@v3.27.0 - Adding test tagging mechanism #globalconfig 

### PR DESCRIPTION
**Pull request by siggerzz**

### CODE REVIEW

**Review by LTurns — DISMISSED** [review]

File: packages/tools/generator-component/generators/app/templates/__package__.json [hunk 1]
```diff
@@ -33,9 +33,9 @@
     "lint:fix": "yarn lint --fix"<% if(config.isComponent) { %>,
     "lint:style": "vue-cli-service lint:style"<% } %>,
```
Comment (LTurns) [inline packages/tools/generator-component/generators/app/templates/__package__.json:34]: is there anything else we need to change in the generator? Is it worth adding the `@browserstack` tag?

Comment (siggerzz) [reply packages/tools/generator-component/generators/app/templates/__package__.json:34]: I did think this, but would require changing more files. Happy to do this in another PR once we've got a consensus on what tests should be run against BS

**Comment by ashleynolan** [comment]
Looks like this PR should use the [globalconfig](https://justeat.github.io/fozzie-components/@justeat/storybook/index.html?path=/story/documentation-getting-started-contributing--page#globalconfig) flag rather than the trivial flag (so that it runs danger checks on the root of the mon-repo then, but not packages).

**Comment by siggerzz** [comment]
> Looks like this PR should use the [globalconfig](https://justeat.github.io/fozzie-components/@justeat/storybook/index.html?path=/story/documentation-getting-started-contributing--page#globalconfig) flag rather than the trivial flag (so that it runs danger checks on the root of the mon-repo then, but not packages).

My bad! I always forget 🤦🏼

**Review by oliversweb — DISMISSED** [review]

**Review by oliversweb — APPROVED** [review]

**Review by LTurns — APPROVED** [review]

**Pull request merged**